### PR TITLE
Localize FXIOS-5651 [v112] Add settings page for notifications

### PR DESCRIPTION
**Pull request by thatswinnie**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-5651)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/13043)

### Description
This updates the string for the notification settings.

### Pull requests checks where applicable
- [x] Fill in the three TODOs above (tickets number and description of your work)
- [x] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
~~- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)~~
~~- [ ] Unit tests written and passing~~
~~- [ ] Documentation / comments for complex code and public methods~~

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1274,35 +1274,17 @@ extension String {
...
                 tableName: "Settings",
-                value: "Allow All Notifications",
-                comment: "This is the title for the setting that toggles All Notifications feature in the settings menu under the Notifications section."
+                value: "Sync",
+                comment: "This is the title for the setting that toggles Sync related notifications in the settings menu under the Notifications section."
             )
-            public static let TabsNotificationsTitle = MZLocalizedString(
-                "Settings.Notifications.TabsNotificationsTitle",
+            public static let SyncNotificationsStatus = MZLocalizedString(
+                "Settings.Notifications.SyncNotificationsStatus",
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1284]: I think we can add a `.v112` to the string keys as well.

**Review by nishant2718 — APPROVED** [review]

File: Client/Frontend/Strings.swift (hunk 1 above)
Comment (thatswinnie) [reply Client/Frontend/Strings.swift:1284]: Oh yes, absolutely. Totally missed that. Thanks!

**Comment by mte-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Edited 2 files</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Created 0 files</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against aeaaa2753e92c160645b5f9568e2f40f2cc04357
</p>

File: Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -1269,61 +1269,43 @@ extension String {
...
             public static let TurnOnNotificationsTitle = MZLocalizedString(
-                "Settings.Notifications.TurnOnNotificationsTitle",
+                "Settings.Notifications.TurnOnNotificationsTitle.v112",
                 tableName: "Settings",
                 value: "Turn on Notifications",
                 comment: "This is the title informing the user needs to turn on notifications in iOS Settings."
             )
             public static let TurnOnNotificationsMessage = MZLocalizedString(
-                "Settings.Notifications.TurnOnNotificationsTitle",
+                "Settings.Notifications.TurnOnNotificationsTitle.v112",
```
Comment (lmarceau) [inline Client/Frontend/Strings.swift:1308]: Those two strings have the same keys.

Comment (thatswinnie) [reply Client/Frontend/Strings.swift:1308]: Great catch. Fixing!

**Comment by thatswinnie** [comment]
Build before the last small commit was green. Not running it again.
Screenshot 2023-03-09 at 16 54 31

**Pull request merged**